File: Assignment/Module-1(fundamentals/p10.cpp
#include<stdio.h>
main()
{
	double num;
	printf("Enter a number:");
	scanf("%lf",&num);
	if(num<=0.0)
	{
		if(num==0.0)
		printf("you entered 0.");
		else
		printf("you entered a negative number.");
	}
	else
	printf("you entered a positive number.");
}

File: Assignment/flutter dart assignment module-1/p10.dart
import 'dart:io';

void main() {
  double number;

  stdout.write("Enter a number: ");
  number = double.parse(stdin.readLineSync()!);

  if (number > 0) {
    print("$number is a positive number.");
  } else if (number < 0) {
    print("$number is a negative number.");
  } else {
    print("$number is zero.");
  }
}
